landoapi/repos.py

```diff
@@ -88,6 +88,13 @@ def __post_init__(self):
         if not self.short_name:
             self.short_name = self.tree
 
+    @property
+    def formatting(self) -> bool:
```

Comment:
For consistency, I would suggest renaming this to `has_autoformat_enabled`, or something similar.